src/content/tutorials/configure-your-cdn/index.md

```diff
@@ -0,0 +1,271 @@
+
+```javascript
+---
+filename: index.js
+highlight: [3, 4, 5, 6, 7]
+---
+async function handleRequest(event) {
+  if (event.request.method === 'GET') {
+    let response = await serveAsset(event)
+    if (response.status > 399) {
+      response = new Response(response.statusText, { status: response.status })
+    }
+    return response
+  } else {
+    return new Response('Method not allowed', { status: 405 })
+  }
+}
+```
+
+And with that, you’re finished writing the code for this tutorial! The final version of your script should like this:
+
+```javascript
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event))
+})
+
+const BUCKET_NAME = 'hugo-workers'
+const BUCKET_URL = `http://storage.googleapis.com/${BUCKET_NAME}`
+
+async function serveAsset(event) {
+  const url = new URL(event.request.url)
+  const cache = caches.default
+  let response = await cache.match(event.request)
+
+  if (!response) {
+    response = await fetch(`${BUCKET_URL}${url.pathname}`)
+    const headers = { 'cache-control': 'public, max-age=14400' }
+    response = new Response(response.body, { ...response, headers })
+    event.waitUntil(cache.put(event.request, response.clone()))
+  }
+  return response
+}
+
+async function handleRequest(event) {
+  if (event.request.method === 'GET') {
+    let response = await serveAsset(event)
+    if (response.status > 399) {
+      response = new Response(response.statusText, { status: response.status })
+    }
+    return response
+  } else {
+    return new Response('Method not allowed', { status: 405 })
+  }
+}
+```
+
+## Publish
+
+To make this script available for use, you’ll need to build and publish it to Cloudflare using Wrangler. To do this, we’ll first _build_ the code, and then _publish_ it:
+
+```sh
+---
+header: Build and publish your project
+---
+$ wrangler build
+$ wrangler publish
+```
+
+After deploying your project, open up your browser to test retrieving your assets! For instance, if your Workers project is deployed to the route `myassets.com`, going to the URL `myassets.com/faces/1.jpg` should show an uploaded asset. In addition, you can inspect the request in your browser to confirm that it’s being cached. Cloudflare’s CDN will send a `cf-cache-status` header, with `HIT` or `MISS`, to indicate whether the content was a “cache hit” or not:
+
+![Result](./media/result.png)
+
+## Resources
+
+In this tutorial, you built and published a serverless function to Cloudflare Workers for serving assets from cloud storage. If you’d like to see the full source code for this application, visit the [repo on GitHub](https://github.com/signalnerve/assets-on-workers).
+
+
+If you want to get started building your own projects, check out the quick-start templates we’ve provided in our [Template Gallery](/templates).
```

Comment:
Update to new ending seen in other new l18n tutorial (Template Gallery): 

https://github.com/cloudflare/workers-docs-engine/blob/5b0395e72f3c4d7d052fda7a4d8fc4d1b3946490/src/content/tutorials/localize-a-website/index.md#L281